x/evm/keeper/grpc_query_test.go

```diff
@@ -25,10 +25,10 @@ import (
 const invalidAddress = "0x0000"
 
 // expGasConsumed is the gas consumed in traceTx setup (GetProposerAddr + CalculateBaseFee)
-const expGasConsumed = 7700
+const expGasConsumed = 7436
```

Comment:
Why did this change?

Reply:
Anytime we add a static precompile the gas consumption changes, most probably because we are loading more precompiles.